neural_compressor/torch/quantization/config.py

```diff
@@ -923,7 +923,7 @@ def register_supported_configs(cls) -> List[OperatorConfig]:
     def get_model_info(model: torch.nn.Module, example_inputs) -> List[Tuple[str, Callable]]:
         from neural_compressor.torch.algorithms.smooth_quant import get_quantizable_ops_recursively
 
-        model_info, _, _, _ = get_quantizable_ops_recursively(model, example_inputs=example_inputs)
+        model_info, _, _, _, _ = get_quantizable_ops_recursively(model, example_inputs=example_inputs)
```

Comment:
I'm a little confused about line 821 and 926, which use different position to return model_info.

Reply:
This op_name mapping feature has not been added to SmoothQuant yet (might be added later), thus using `get_quantizable_ops_recursively` to return two different formats of model info: converted pt format for static quant, and the original ipex format for sq for now.